src/openrct2-ui/windows/Finances.cpp

```diff
@@ -482,15 +482,23 @@ class FinancesWindow final : public Window
             case WIDX_LOAN_INCREASE:
             {
                 auto newLoan = gBankLoan + 1000.00_GBP;
+                if (gBankLoan < gMaxBankLoan)
+                {
+                    newLoan = std::min(gMaxBankLoan, newLoan);
+                }
```

Comment:
Isn't this just:
```suggestion
                auto newLoan = std::min(gBankLoan + 1000.00_GBP, gMaxBankLoan);
```

Reply:
Not at all. If the bank loan is already maxed out or even higher, the clamping will not be triggered in my solution and instead yield an error message ("The bank refuses to increase your loan!"). In your solution, the error message is suppressed.

Reply:
Should I add a clarifying comment to the code?

Reply:
Ah, I see. Yes, I think a comment would come in handy